apispec/ext/marshmallow/swagger.py

```diff
@@ -41,6 +41,32 @@
     marshmallow.fields.List: ('array', None),
 }
 
+CUSTOM_FIELD_MAPPING_ATTR = '__swagger_field_mapping'
+
+
+def map_to_swagger_field(*args):
```

Comment:
It may be technically optional, but consider that the type for a datetime is `{type: string, format: date-time}` or something like that. I assume that without the format, you're gonna have a bad time.